source/javascripts/components/WorkflowConfigPanel/WorkflowConfigPanel.tsx

```diff
@@ -0,0 +1,48 @@
+import { FormProvider, useForm } from 'react-hook-form';
+import { Tab, TabList, TabPanels, Tabs } from '@bitrise/bitkit';
+import { PartialDeep } from 'type-fest';
+import Header from './components/Header';
+import { FormValues } from './WorkflowConfigPanel.types';
+import PropertiesTabPanel, { PROPERTIES_TAB_ID } from './components/PropertiesTabPanel';
+import ConfigurationTabPanel, { CONFIGURATION_TAB_ID } from './components/ConfigurationTabPanel';
+
+type Props = {
+  workflowId: string;
+  defaultValues?: PartialDeep<Omit<FormValues, 'workflowId'>>;
+  onChange: (data: FormValues) => void;
+};
+
+const WorkflowConfigPanel = ({ workflowId, defaultValues, onChange }: Props) => {
+  const form = useForm<FormValues>({
+    defaultValues: {
+      workflowId,
+      ...defaultValues,
+    },
+  });
+
+  const handleSubmit = form.handleSubmit((data, e) => {
+    e?.preventDefault();
+    e?.stopPropagation();
```

Comment:
[Q]: Aren't these **called automatically** by `handleSubmit`?

Reply:
It seems, the preventDefault method is invoked: https://github.com/react-hook-form/react-hook-form/blob/master/src/logic/createFormControl.ts#L1127